includes/Core/Authentication/Exception/Google_Proxy_Code_Exception.php

```diff
@@ -22,6 +22,15 @@
  */
 class Google_Proxy_Code_Exception extends Exception {
 
+	/**
+	 * Temporary code for an undelegated proxy token.
+	 *
+	 * @since 1.0.0
```

Comment:
When we are introducing something new we write `n.e.x.t` as the version number, which is then updated to use the version number of the plugin it will get shipped with.

Where are these version numbers coming from? Are they when we started to use these properties dynamically? If so it'd be better write that and then a new `n.e.x.t` annotation highlighting when we started explicitly declaring them.

Though really it's probably fine to just write `n.e.x.t` here and mention they were dynamic before this version. 🙂 

```suggestion
	 * @since n.e.x.t
```

Reply:
@tofumatt Yes I was thinking about that to include n.e.x.t, but since these properties were actually dynamically included in previous versions, I mirrored these version. But you have a point, I will put version they were declared and mention they were dynamic before that. Thank you!